war/src/main/less/base/style.less

```diff
@@ -1566,6 +1567,7 @@ svg.icon-sm {
 svg.icon-md {
     width: 24px;
     height: 24px;
+    vertical-align: top;
```

Comment:
why top?  to me this seems like it should be middle?

see https://issues.jenkins.io/browse/JENKINS-68356  changing it to middle actually fixes that bug.  The screenshots in the PR description also look like they are aligned in the middle - and the generic icon is using middle -> https://github.com/jenkinsci/jenkins/pull/6341/files#diff-300f384d47e9aad264a350440de6adfe3cbb8a33c6925206700382807f950b6cR1609